### DISCUSSION TITLE
[Feature] Select multiple points on a map, then delete

### DISCUSSION THREAD

**Discussion by HStep20** [Freika/dawarich#869] [Ideas]
Sometimes, a phone GPS can freak out and record a lot of noise when there is none - This is especially true when you may be indoors/underground. During a recent trip, I visited an place that had quite a bit of underground/indoors areas, and on account of that I have a lot of bad points (see image)
![Screenshot 2024-08-21 190711](https://github.com/user-attachments/assets/2027b6eb-a197-456c-ac23-9a6d080f44e8)

That weirdly straight North-West line spans about 15 miles, while I spent the entire time in the red circled areas. 

Since points can be selected individually on the map, Id love to see a way to select points similar to how the 'areas' part of the map works, and the ability to delete all selected points at once. Its impossible to tell which of these points is which in the table log, since accurate points were recorded while I was there, meaning some sort of visual selector (or manually going one by one for 100+ points) is the only way to clean this up right now

**Comment by HStep20** [comment]
Hopefully bumping this, as the 'area selection' exists now. You can select data in an area, but it doesnt seem like theres a way to act on that selection. Having an option to 'delete selected points' from this would be perfect for cleaning up errant data from bad gps signal (flights, underground, etc)

![image](https://github.com/user-attachments/assets/9c1aec3a-1c55-442d-8c3e-3d1dd906515a)

**Reply by TheDev42** [reply]
i agree, its kind of a useless feature at the moment
